tests/mask.py

```diff
@@ -99,10 +99,10 @@ def setUp(self):
         except Exception:
             self.maskFile = None
         else:
-            if True:
-                self.maskFile = os.path.join(dataDir, "small_MI.fits")
-            else:
-                self.maskFile = os.path.join(dataDir, "871034p_1_MI.fits")
+            self.maskFile = os.path.join(dataDir, "small_MI.fits")
+            # Above mask looks like this, possibly left-shifted by some number of mask planes.
```

Comment:
Ticket was https://jira.lsstcorp.org/browse/DM-609 and there are some duplicates as people keep wanting the tests to run and skip rather than skipping the entire test suite.